src/components/CurrencyInputPanel/index.js

```diff
@@ -143,14 +148,16 @@ class CurrencyInputPanel extends Component {
   };
 
   validate = (balance) => {
-    const { value, addError, removeError, errors } = this.props;
+    const { value, addError, removeError, errors, validateBalance } = this.props;
```

Comment:
how is `validateBalance` being passed here?

Also, `validateBalance` implies that it is a `function`. should this be called `isBalanceValid`?

Reply:
`validateBalance` is telling the inputpanel whether or not it should even validate the input being greater than the balance. we don't always want it to validate that (the Output shouldn't validate it) so I added this prop

Reply:
I just changed the name from `validateBalance` to `shouldValidateBalance` to make the purpose more clear